lib/actuators/include/ContServo.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "Servo.h"
+
+class ContServo : public Servo {
+ public:
+  ContServo(PinName pin);
+  ContServo(PinName pin, float max_speed);
+  ContServo(PinName pin, float max_speed, std::chrono::duration<float> max_pulse,
+            std::chrono::duration<float> min_pulse);
+
+  /** Set the maximum speed of the servo
+   *
+   * @param max_speed The maximum speed of the servo
+   */
+  void setMaxSpeed(float max_speed);
```

Comment:
do we need to call this function everytime we create an object? if so, max speed belongs in the constructor. this is RAII

Reply:
Ok, I assume the same applies to setPeriod() right?

Reply:
yes also that